WORKSPACE

```diff
@@ -96,11 +96,22 @@ http_archive(
 )
 
 http_archive(
-    name = "io_bazel_rules_docker",
-    sha256 = "b1e80761a8a8243d03ebca8845e9cc1ba6c82ce7c5179ce2b295cd36f7e394bf",
-    urls = ["https://github.com/bazelbuild/rules_docker/releases/download/v0.25.0/rules_docker-v0.25.0.tar.gz"],
+    name = "rules_distroless",
```

Comment:
how does https://github.com/GoogleContainerTools/rules_distroless act as an alternative?

Reply:
rules_docker is deprecated and is no longer supported.

rules_distroless offers some of the functionality that rules_docker supported. Specifically, it replaced the passwd.tar generation in tools/BUILD.bazel